application.js

```diff
@@ -0,0 +1,16 @@
+var shapeRegistry = [];
+var test = new Circle(100, 150, 50, 'brown', 'pink', 30);
+shapeRegistry.push(test);
+
+setTimeout(function() {
```

Comment:
also of note: I don't know if we'll get much from inheritance. There appears to be little in common between the shapes.